docs/user_guides/benchmarking/run.rst

```diff
@@ -167,6 +171,23 @@ For example, if we want to evaluate our synthesizer function on only the
 
    In [15]: scores = sdgym.run(my_synthesizer_function, datasets=datasets)
 
+Running on Private Datasets
+~~~~~~~~
+
+If we want to run sdgym on datasets in a private s3 bucket, we can pass in
```

Comment:
I would rephrase this a bit to mention something like "in a private S3 bucket for which we do not have system-wide credentials configured", and link the corresponding part of the sentence to the AWS documentation that explains how to configure `.aws/credentials` or set environment variables.